content/api/transmissions.apib

```diff
@@ -634,8 +638,6 @@ Only transmissions which are scheduled for future generation may be deleted.
 
 ### Delete Scheduled Transmissions By Campaign ID [DELETE /v1/transmissions{?campaign_id}]
 
-<Banner status="info">Enterprise only, account-specific configuration option.</Banner>
-
 Delete all transmissions in a campaign.
 
```

Comment:
Please note that this will delete any messages with this campaign_id that remain to be delivered, whether or not they were originally scheduled. If the same campaign_id was used for messages that had temporary failed and are waiting on the delayed queue to be retried, then these messages will also be deleted.